pkg/kubelet/dockershim/helpers_windows.go

```diff
@@ -40,18 +47,41 @@ func (ds *dockerService) getSecurityOpts(seccompProfile string, separator rune)
 	return nil, nil
 }
 
+func shouldIsolatedByHyperV(annotations map[string]string) bool {
+	if v, ok := annotations[hypervIsolationAnnotationKey]; ok {
+		if v == hypervIsolation {
+			return true
+		}
+	}
+
+	return false
+}
+
+// applyExperimentalCreateConfig applys experimental configures from sandbox annotations.
+func applyExperimentalCreateConfig(createConfig *dockertypes.ContainerCreateConfig, annotations map[string]string) {
+	if shouldIsolatedByHyperV(annotations) {
+		createConfig.HostConfig.Isolation = hypervIsolation
+
+		if networkMode := os.Getenv("CONTAINER_NETWORK"); networkMode == "" {
+			createConfig.HostConfig.NetworkMode = dockercontainer.NetworkMode("none")
```

Comment:
Does this mean that hyperV isolated containers will not have any networking by default?

Reply:
no, hyperV container's network will be set by CNI plugins